src/infrastructure/namespace.yml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: todoapp

```

Comment:
Still the wrong namespace name. Namespace names have to be aligned with the one you use in your k8s manifests.
They are different now but should be the same.